src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -19,7 +19,8 @@ const MetaTrader = (() => {
 
     const onLoad = () => {
         BinarySocket.send({ statement: 1, limit: 1 });
-        BinarySocket.wait('landing_company', 'get_account_status', 'statement').then(async () => {
+        BinarySocket.send({ trading_platform_accounts: 1, platform: 'dxtrade' });
+        BinarySocket.wait('landing_company', 'get_account_status', 'statement', 'trading_platform_accounts').then(async () => {
```

Comment:
Having requested a Deriv X accounts list, I get it from `trading_platform_accounts`.